apps/src/templates/levelSummary/FreeResponseResponses.jsx

```diff
@@ -1,18 +1,27 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
+import DCDO from '@cdo/apps/dcdo';
+
 import ResponseMenuDropdown from './ResponseMenuDropdown';
 
 import styles from './summary.module.scss';
 
-const FreeResponseResponses = ({responses}) => {
+const FreeResponseResponses = ({responses, showStudentNames}) => {
   return (
     <div className={styles.studentResponsesColumns}>
       {responses.map(response => (
-        <div key={response.user_id} className={styles.studentAnswer}>
-          <div className={styles.studentAnswerInterior}>
-            <p>{response.text}</p>
-            <ResponseMenuDropdown response={response} />
+        <div key={response.user_id} className={styles.studentResponseBlock}>
+          <div className={styles.studentAnswer}>
+            <div className={styles.studentAnswerInterior}>
+              <p>{response.text}</p>
+              <ResponseMenuDropdown response={response} />
+            </div>
+            {DCDO.get('cfu-show-names-enabled', false) && (
```

Comment:
Do we want to put this under the same or different DCDO flag to pinning? I understand I named mine incorrectly to show names, but we may just want/need the one. What do you think?

Reply:
Ah yeah, I was thinking the same flag but when I saw your flag name I thought maybe different... I could see them getting done at slightly different times and then us wanting two different flags.  

That being said, that does make for more variations of testing... let's do one flag.  I will remove mine and just stick with yours (with its current name)